csv-validation-tests/src/main/scala/Column.scala

```diff
@@ -0,0 +1,22 @@
+package typeDefinitions
+case class Column(
+                 aboutUrl: String,
+                 cells: List[Cell],
+                 dataType: String, // revisit, (the expected datatype for the values of cells in this column, as defined in [tabular-metadata])
```

Comment:
Time for generics?

Reply:
In the initial stages of the project, I was creating type classes which I thought would be required in the future. This is not in use currently and is dead code at the moment. I never removed these classes hoping it can be used in future.